src/components/ADempiere/FieldDefinition/useFieldDefinition.js

```diff
@@ -178,8 +178,12 @@ export default function useFieldDefinition({ fieldMetadata, containerManager })
    * @returns promisse with object = { value, defaultValue, uuid, id }
    */
   function getDefaultValueFromServer() {
+    console.log({
+      fieldMetadata: fieldMetadata
+    })
     if (containerManager && containerManager.getDefaultValue) {
       return containerManager.getDefaultValue({
+        ...fieldMetadata,
```

Comment:
remove this unnecesary metadata.

Reply:
[d816b1a](https://github.com/solop-develop/frontend-core/pull/1501/commits/d816b1a65c8692e247c471fd63dfccf76a6e0d58)

Reply:
remove this unnecesary metadata.